Makefile

```diff
@@ -85,16 +85,16 @@ TESTS=.
 TE_PACKAGES=$(shell $(GO) list ./...)
 
 # Plugins Packages
-PLUGIN_PACKAGES?=mattermost-plugin-zoom-v1.3.0
-PLUGIN_PACKAGES += mattermost-plugin-autolink-v1.1.2
+PLUGIN_PACKAGES?=mattermost-plugin-zoom-v1.3.1
+PLUGIN_PACKAGES += mattermost-plugin-autolink-v1.2.0
 PLUGIN_PACKAGES += mattermost-plugin-nps-v1.0.4
-PLUGIN_PACKAGES += mattermost-plugin-custom-attributes-v1.0.2
-PLUGIN_PACKAGES += mattermost-plugin-github-v0.11.0
+PLUGIN_PACKAGES += mattermost-plugin-custom-attributes-v1.2.0
+PLUGIN_PACKAGES += mattermost-plugin-github-v0.14.0
```

Comment:
This should be `v1.0.0`

Reply:
Sorry, I took the versions directly from the Jira ticket.